### PR TITLE
dabangapp.com - Incorrect Blocking

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [X] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://dabangapp.com/room/62b3e8a06045c068354322a9

#### How to reproduce

Click the highlighted element of the following screenshot:

<details>
<summary>Screenshot</summary>

Screenshot6767

</details>

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by zloyden — COMMENTED** [review]
https://dabangapp.com/room/62b3e8a06045c068354322a9 this link does not exist.

**Comment by piquark6046** [comment]
@zloyden 
Please use the following URL:
https://dabangapp.com/room/628db6cf2f3642527edb98cf

File: SpywareFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -306,7 +306,7 @@ extrahop.com#%#//scriptlet('prevent-setTimeout', 'ad blocker')
 ! www.satofull.jp images not shown
 @@||rtoaster.jp/Rtoaster.js$domain=satofull.jp
 ! https://github.com/AdguardTeam/AdguardFilters/issues/111052
-@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr
+@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr|dabangapp.com|kyobobook.co.kr
```
Comment (zloyden) [inline SpywareFilter/sections/whitelist.txt:309]: Ok, but why do you need an exception for `|kyobobook.co.kr`?

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -2853,7 +2853,7 @@ dreamprogs.net##td[width="88"][height="31"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 ||play.google.com/log?
 !+ NOT_PLATFORM(ios, ext_android_cb)
-||naver.net/wcslog.js$domain=~kyobobook.co.kr
+||naver.net/wcslog.js
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:2856]: Please also explain this.

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:2856]: I think what allowing the request that blocks EasyList by default in AdguardFilters is better.

File: SpywareFilter/sections/whitelist.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/whitelist.txt:309]: I checked on Mozilla Firefox Dev 102.0b9 with AdGuard Browser Extension 4.0.181.

**Review by zloyden — APPROVED** [review]

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply SpywareFilter/sections/specific.txt:2856]: Ok, I got it.

**Pull request merged**